Apps/Sandcastle/gallery/3D Tiles Point Cloud Styling.html

```diff
@@ -129,6 +129,11 @@
     color : "rgb(${POSITION}[0] * 255, ${POSITION}[1] * 255, ${POSITION}[2] * 255)"
 });
 
+addStyle('Style point size', {
+    color : "color('red')",
+    size : "${temperature} * 10"
```

Comment:
My thinking was a generic name would be good for other sizeable things thing billboards. If fine with either way.